cmd/run.go

```diff
@@ -34,7 +36,8 @@ var (
 )
 
 func init() {
-	runCmd.Flags().StringSliceVarP(&nodeIPs, "node-ips", "n", []string{}, "List of remote hosts")
+	runCmd.Flags().StringSliceVarP(&nodeIPs, "node-ips", "n", []string{}, "Comma-delimited list of remote hosts")
```

Comment:
This is where things get a little sticky. It's pretty dirty to relate public-to-private IPs based on the ordinality of separate lists. We can reformat the way we ingest node IPs (e.g. support `--node-ips "<pub1:priv1>,<pub2:priv2>"` but still allow `--node-ips "<lone1>,<lone2>` ) or move to a file-based config file.

I think we can get by w/o using a config file for now, but we may have to if we end up adding any more complex or compound input values. FWIW, integrating a config file is easy.

Reply:
I completely agree, was even going to do exactly that, but a misinterpretation (before I really read through hosts.go, I had the mistaken impression you could pass ip:port,ip:port) got me off that path. I'll switch it to this.